tools-local/unix/acquire-tool.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
+
+# Downloads the specified tool.
+# Download URL is read from the .toolversions file.
+
+if [ -z "$1" ]; then
+    echo "Argument passed as tool name is empty. Please provide a non-empty string."
+    exit 1
+fi
+
+toolName="$1"
+lowercaseToolName="$(echo "$toolName" | awk '{print tolower($0)}')"
```

Comment:
If we can stick to lowercase tool names in toolversions and folder names like "cmake", "clang" then, lowercase'ing here is not required.

---
In reply to: [107791461](https://github.com/ravimeda/corefx/pull/2#discussion_r107791461) [](ancestors = 107791461)